spec/expectations/grid_form_odk.xml

```diff
@@ -5,12 +5,20 @@
     <model>
       <instance>
         <data id="1" version="1">
+          <name2/>
+          <hint2/>
```

Comment:
Perhaps `group-hint2` or `grp-hint2`? hint alone is ambiguous, from a readability perspective.